examples/docs_snippets/docs_snippets/guides/dg/workspace/2-dg-scaffold-workspace.txt

```diff
@@ -1,4 +1,4 @@
-dg scaffold project --python-environment uv_managed projects/project-1
+create-dagster project --python-environment uv_managed projects/project-1
```

Comment:
@schrockn here is an example. These are code snippets that are pulled in in various pages - the docs themselves haven't changed because the only thing that has changed is the command in the code snippet.

So we could either:
- leave all of these as create-dagster
- change them all to uvx create-dagster
- add uv/non-uv choose your own adventure tabs to every page where a project gets scaffolded